content/doc/developer/forms/jelly-form-controls.adoc

```diff
@@ -121,3 +133,30 @@ Then, use an `<f:entry>` tag to enclose the normal select tag.
    <f:select />
 </f:entry>
 ----
+
+== Inline help
+
+* jelly definition link:https://github.com/jenkinsci/jenkins/blob/master/core/src/main/resources/lib/form/helpLink.jelly[form/helpLink.jelly]
+
+Most of form elements allows to have an inline help generated by adding a small html file and following conventions.
+
+For example <f:entry> tag, uses the field name to find the
+
+Given a file in src/main/resources/myForm.jelly with this content:
+
+[source, xml]
+----
+<f:entry title="Visible" field="isVisible">
```

Comment:
Not the best example since conventionally the field would be called `visible` corresponding to a JavaBeans getter/setter `setVisible` / `isVisible`.

Reply:
I copied my morning example from https://github.com/jenkinsci/promoted-builds-plugin/blob/master/src/main/resources/hudson/plugins/promoted_builds/PromotionProcess/help-isVisible.html

I can change it to something better, like name

Reply:
changed